Packs/Code42/Integrations/Code42/README.md

```diff
@@ -111,21 +111,21 @@ Retrieve alert details by alert ID
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| id | The alert ID to retrieve. Alert IDs are associated with alerts that are fetched via fetch-incidents. | Required | 
+| id | The alert ID to retrieve. Alert IDs are associated with alerts that are fetched via fetch-incidents. | Required |
```

Comment:
It looks like some tooling I have removed a bunch of trailing whitespace here, sorry. Only the entries at the bottom for the new commands should be different.